client/src/document/index.tsx

```diff
@@ -263,7 +263,7 @@ function LoadingError({ error }) {
         </p>
       )}
       <p>
-        <a href=".">Try reloading the page</a>
+        <a href="?">Try reloading the page</a>
```

Comment:
I have never seen this. Why would a `?` be preferred over a `.`?

Reply:
The `.` references the directory, whereas the `?` references the current file with an empty query.

This bugged me while working on the BCD tables, because I got regular 504s, and this `Try reloading the page` link didn't reload the current article, but instead opened the parent article.